codegen/templates/endpoint.tmpl

```diff
@@ -50,20 +52,25 @@ func Handle{{title .Name}}Request(
 	{{end}}
 
 	headers := map[string]string{}
+	// TODO(sindelar): Add optional headers in addition to required.
+	for _, k := range {{.Headers | printf "%#v" }} {
```

Comment:
I don't think we want all of them. Content-length definitely breaks downstream requests, could potentially content-type.